eng/pipelines/templates/jobs/sign-and-pack.yml

```diff
@@ -1,10 +1,8 @@
 parameters:
-- name: ServerName
-  type: string
-- name: PackageDocker
-  type: boolean
-- name: PackageVSIX
-  type: boolean
+  - name: ServerName
```

Comment:
It would be nice if we can keep sign-and-pack working against all servers in build_info.json.  If verify_packages is the only place we filter down to a single server, lets just remove that filter